Build throwing unused variables warnings/errors on non-Windows platforms

third_party/vulkan_loader/loader/loader.c:7064:14: error: unused variable 'sorted_alloc' [-Werror,-Wunused-variable]
    uint32_t sorted_alloc = 0;
             ^
third_party/vulkan_loader/loader/loader.c:7065:29: error: unused variable 'icd_term' [-Werror,-Wunused-variable]
    struct loader_icd_term* icd_term = NULL;
                            ^
2 errors generated.

--- a/loader/loader.c
+++ b/loader/loader.c
@@ -7061,10 +7061,10 @@ struct LoaderSortedPhysicalDevice {
 VkResult ReadSortedPhysicalDevices(struct loader_instance *inst, struct LoaderSortedPhysicalDevice **sorted_devices, uint32_t* sorted_count)
 {
     VkResult res = VK_SUCCESS;
-    uint32_t sorted_alloc = 0;
-    struct loader_icd_term* icd_term = NULL;
 
 #if defined(_WIN32)
+    uint32_t sorted_alloc = 0;
+    struct loader_icd_term* icd_term = NULL;
     IDXGIFactory6* dxgi_factory = NULL;
     HRESULT hres = fpCreateDXGIFactory1(&IID_IDXGIFactory6, &dxgi_factory);
     if (hres != S_OK) {

--- a/loader/loader.c
+++ b/loader/loader.c
@@ -7064,7 +7064,7 @@ VkResult ReadSortedPhysicalDevices(struct loader_instance *inst, struct LoaderSo
 
 #if defined(_WIN32)
     uint32_t sorted_alloc = 0;
-    struct loader_icd_term* icd_term = NULL;
+    struct loader_icd_term *icd_term = NULL;
     IDXGIFactory6* dxgi_factory = NULL;
     HRESULT hres = fpCreateDXGIFactory1(&IID_IDXGIFactory6, &dxgi_factory);
     if (hres != S_OK) {

